manifests/install.pp

```diff
@@ -0,0 +1,27 @@
+# @summary Installs cron
```

Comment:
I saw some pieces of this class tested in the cron class, but I don't understand why it doesn't have its own test.

Reply:
@lnemsick-simp  I made the changes to this module.  If you are happy I will push it.